src/lib/coda_metrics/coda_metrics.ml

```diff
@@ -213,6 +213,41 @@ module Network = struct
   (* TODO: track non gossip RPC messages as well *)
 end
 
+module Snark_work = struct
+  let subsystem = "Snark_work"
+
+  let snark_work_received_gossip : Counter.t =
+    let help = "# of snark work received from peers" in
```

Comment:
Is `work` synonymous with `job` in the classic definition of the word? 

i.e. would it be better to refer to these as "Snark Jobs" or "Snark Work Jobs"?

Reply:
This term is kind of overloaded. There's basically work that needs to be done and then there's work that's done. I tend to use `job` for the former and completed work/work for the latter. So scan_state_snark_jobs are the ones that are yet to be done/purchased. snark_work_assigned_rpc in that case should be snark_job_assigned_rpc